tools/scripts/generate_binary_build_matrix.py

```diff
@@ -133,13 +133,13 @@ def translate_desired_cuda(gpu_arch_type: str, gpu_arch_version: str) -> str:
 def list_without(in_list: List[str], without: List[str]) -> List[str]:
     return [item for item in in_list if item not in without]
 
-def get_conda_install_command(channel: str, gpu_arch_type: str, arch_version: str) -> str:
+def get_conda_install_command(channel: str, gpu_arch_type: str, arch_version: str, os: str) -> str:
     conda_channels = "-c pytorch" if channel == RELEASE else f"-c pytorch-{channel}"
-
+    conda_package_type = ""
     if gpu_arch_type == "cuda":
         conda_package_type = f"pytorch-cuda={arch_version}"
         conda_channels = f"{conda_channels} -c nvidia"
-    else:
+    elif os not in ("macos", "macos-arm64"):
```

Comment:
"elif" requires "else"?

Reply:
I initialize conda_package_type = "" in line 138, so no else needed. By default its empty

Reply:
Technically from language point of view, else is not required. Please refer to: https://stackoverflow.com/questions/27916891/does-elif-must-followed-by-else-in-python-as-a-common-rule